Update URLs and examples for new sandbox

daliboz opened the pull request:
Update all existing resources to point to valid examples in the new sandbox. Update DSTU 2 documentation to reference the new sandbox.

MaxPhilips commented on lib/resources/example_json/dstu2_examples_care_plan.rb:
```diff
@@ -49,16 +49,16 @@ module Resources
+          "fullUrl": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/CarePlan/7499295",
           "resource": {
             "resourceType": "CarePlan",
             "id": "7499295",
```
These are still the old sandbox ids, I assume it was left on purpose because Tim Peters' CarePlans in FHIRPLAY are goofy "XSL FO Document" ones.

Looks like patients 1316020 and 1316026 have some more 'real' CarePlans if we wanted to update these examples to ids that exist in the domain.

While this file is being updated, can you find/replace `&gt;` with `&amp;gt;`?  Looks like I overlooked that when writing this doc.

daliboz replied:
If the URL still worked, I didn't update it. I didn't check to see if it was the exact same data or not :) I can update to more realistic data.

MaxPhilips commented:
Most of the examples now have the squiggly bracket to start an array entry on the same line as the square bracket, but a few examples kept the current formatting of putting the squiggly bracket on a newline.  I prefer the second way, and we should probably keep everything formatted consistently in whichever style.

dennispatterson commented on content/dstu2.md:
```diff
@@ -388,11 +390,11 @@ follow these Link header values instead of constructing your own URLs.
         {
           "relation": "self",
-          "url": "https://fhir-open.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Patient?name=Jones&start=0&_count=20"
+          "url": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient?name=Jones&start=0&_count=20"
```
This is the syntax for cloud patient search, which isn't present in FHIRPlay.  Madhur/Bradley may be able to give you local patient search examples that will work once that lands on 3/1.  Or, since this example isn't tied to a resource you could use a different one.

dennispatterson commented on content/dstu2/allergy-intolerance.md:
```diff
@@ -84,7 +84,7 @@ To successfully POST an allergy, the following headers must be provided. Allergy
     cache-control → no-cache
     etag → W/"35502191"
-    location → https://fhir-ehr.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/AllergyIntolerance/35502191
+    location → https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/AllergyIntolerance/35502191
```
Is the goal of this PR just to update the data source id and comb back through later to fix the ids, or should all of these urls work after this PR?  I know there's been some [frustration expressed](https://groups.google.com/d/msg/cerner-fhir-developers/YdQuAIzPhfw/0CVrqXRECAAJ) at our examples.

daliboz replied:
The goal was to fix the URLs. It looks like I missed this one and the patient search

dennispatterson commented on content/dstu2/condition.md:
```diff
@@ -78,7 +78,7 @@ Create Problem Example:
     cache-control → no-cache
     etag → W/"4736135"
-    location → https://fhir-ehr.sandboxcerner.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/Condition/p4736135
+    location → https://fhir-ehr.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Condition/p4736135
```
404 for this guy and the one on line 109

daliboz commented:
Spoke with Max offline about json formatting. Came to the agreement we wouldn't necessarily apply this yet, since we need to be able to tell everyone the "easy" way to get the json formatted with the brackets that way. Will be fixing spacing on a few examples that got out of wack.

daliboz commented:
Updated and rebased off master. The Condition URLs and examples were updated to use patients we would want them to use, but you can't get by id yet for condition, so you can search Condition to see the resulting problem or diagnosis

MaxPhilips commented:
+1

cerbs commented:
+1
